arm-customer-insights/2017-01-01/swagger/customer-insights.json

```diff
@@ -3496,16 +3609,7 @@
           "description": "Name of the connector."
         },
         "connectorType": {
-          "type": "string",
-          "enum": [
-            "CRM",
-            "AzureBlob",
-            "Salesforce"
-          ],
-          "x-ms-enum": {
-            "name": "ConnectorTypes",
-            "modelAsString": false
```

Comment:
our service side didn't add new api version for that. So how should I update the version in swagger? Should I just put the this new swagger in a new folder, but still calling the same service api-version 2017-01-01?